src/forms/textInput.test.jsx

```diff
@@ -1,128 +1,117 @@
 import React from 'react';
-import TestUtils from 'react-addons-test-utils';
-import { TestWrapper } from '../utils/testUtils';
 import TextInput from './TextInput';
+import { mount } from 'enzyme';
+import toJson from 'enzyme-to-json';
 
 describe('TextInput', function() {
 	const onChange = jest.fn();
 	const LABEL_TEXT = 'Super Hero',
 		VALUE = 'Batman',
 		NAME_ATTR = 'superhero',
-		MAX_LEN = '20',
+		MAX_LEN = 20,
 		ERROR_TEXT = 'Too wimpy.';
 
 	let textInputComponent,
 		inputEl;
 
+	const formAttrs = {
+		id: NAME_ATTR,
+		maxLength: MAX_LEN,
+		required: true,
+		error: ERROR_TEXT,
+	};
+
 	beforeEach(() => {
-		const formAttrs = {
-			id: NAME_ATTR,
-			maxLength: MAX_LEN,
-			error: ERROR_TEXT,
-			required: true,
-		};
-		textInputComponent = TestUtils.renderIntoDocument(
-			<TestWrapper>
-				<TextInput
-					name={NAME_ATTR}
-					label={LABEL_TEXT}
-					value={VALUE}
-					{...formAttrs}
-				/>
-			</TestWrapper>
+		textInputComponent = mount(
+			<TextInput
+				name={NAME_ATTR}
+				label={LABEL_TEXT}
+				value={VALUE}
+				onChange={onChange}
+				{...formAttrs}
+			/>
 		);
 
-		inputEl = TestUtils.findRenderedDOMComponentWithTag(textInputComponent, 'input');
+		inputEl = textInputComponent.find('input');
 	});
 
 	afterEach(() => {
 		textInputComponent = null;
 		inputEl = null;
 	});
 
-	it('exists', () => {
-		expect(() => TestUtils.findRenderedDOMComponentWithTag(textInputComponent, 'input')).not.toThrow();
+	it('matches snapshot', () => {
+		expect(toJson(textInputComponent)).toMatchSnapshot();
```

Comment:
snapshots can actually be made for any object, so they can be useful if you have a big complex object (e.g. a state tree) that needs to be organized _just so_. The 'rendering' snapshots are special because they serialize the React element into a human-readable format rather than just labeling every prop as a `[Function]` or whatever